third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -1903,20 +1890,10 @@ func TestAccComputeInstance_subnetworkChange(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccComputeInstance_subnetworkChange(suffix, instanceName),
```

Comment:
validating that this looks correct @danawillow

Reply:
👍